File: Exemplos SmartPOS/App_IntentDigitalHUB_Flutter/Elgin_IntentDigitalHUB_SmartPOS_Flutter-font/lib/IntentDigitalHubService/ELGINPAY/Commands/inicia_venda_debito.dart
import 'package:flutter_smartpos/IntentDigitalHubService/ELGINPAY/elginpay_command.dart';

class IniciaVendaDebito extends ElginPayCommand {
  final String _valorTotal;

  IniciaVendaDebito(this._valorTotal) : super('iniciaVendaDebito');

  @override
  get functionParametersJson => {
        'valorTotal': _valorTotal,
      };
}

File: Exemplos SmartPOS/App_IntentDigitalHUB_ReactNative/Elgin_IntentDigitalHUB_SmartPOS_ReactNative-font/src/intentDigitalHubService/ELGINPAY/Commands/IniciaVendaDebito.ts
import {ElginPayCommand} from '../ElginPayCommand';

export class IniciaVendaDebito extends ElginPayCommand {
  readonly valorTotal: string;

  constructor(valorTotal: string) {
    super('iniciaVendaDebito');
    this.valorTotal = valorTotal;
  }

  functionParameters(): object {
    return {
      valorTotal: this.valorTotal,
    };
  }
}
